package.json

```diff
@@ -9,65 +9,63 @@
     "eject": "expo eject",
     "pre": "cp crna-entry.js index.js",
     "prestorybook": "cp storybook-entry.js index.js && rnstl",
-    "storybook": "storybook start",
-    "test": "node node_modules/jest/bin/jest.js --watch",
+    "test": "node node_modules/jest/bin/jest.js",
     "format": "prettier-eslint \"src/**/*.js\"",
     "postinstall": "patch-package"
   },
   "dependencies": {
     "axios": "0.18.0",
-    "date-fns": "1.29.0",
-    "expo": "^31.0.2",
-    "expo-three": "^2.2.2-alpha.1",
+    "date-fns": "1.30.1",
+    "expo": "^31.0.4",
```

Comment:
We should probably lock this version down.  Same with `react-native-walkthrough-tooltip` and `jest-expo`

Reply:
Agreed, will do.